src/components/atoms/Reaction/Reaction.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useCallback } from "react";
+
+import { EmojiReactionType, ReactionData } from "types/reactions";
+
+import "./Reaction.scss";
+
+export interface PropsType {
+  reaction: ReactionData<EmojiReactionType>;
+  reactionClicked?: (emojiReaction: EmojiReactionType) => void;
+}
+
+export const Reaction: React.FC<PropsType> = ({
+  reaction,
+  reactionClicked,
+}) => {
+  const handleReactionClick = useCallback(() => {
+    reactionClicked && reactionClicked(reaction.type);
+  }, [reactionClicked, reaction]);
+
+  return (
+    <button
+      className="Reaction"
+      onClick={handleReactionClick}
+      id={`send-reaction-${reaction.type}`}
```

Comment:
Do you really need this `id`?

Reply:
it was legacy from the `div` version of Reaction. I assume there is no need

Reply:
fixed in d4f3cf3bd46debf17b8f6f9452f37bd409104f32